code/style.css

```diff
@@ -1,4 +1,189 @@
 html {
-  background: #0025ff;
+  background: black;
   color: #fff;
+  font-family: 'Times New Roman', Times, serif;
 }
+
+body {
+  margin: 0px;
+  padding: 0px;
+  width: 100%;
+}
+
+.header {
+  background: rgb(8, 16, 8);
+  width: 100%;
+  padding: 5px 0px;
+  margin: 0px 0px 0px 0px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header:hover {
+  font-size: 20px;
+  background-color: rgb(6, 26, 6);
+}
```

Comment:
Just a heads up, this makes the whole content "jump" a bit since the space for this growth is not allocated. If you give the `.header` a fixed height of e.g. 120px, you'll get rid of some of the jumping. See if you can get rid of all the jumping by doing something similar